src/components/WorkPoolCard.vue

```diff
@@ -1,40 +1,42 @@
 <template>
-  <p-card class="work-pool-card">
-    <div class="work-pool-card__header">
-      <div class="work-pool-card__heading">
-        <div class="work-pool-card__name">
-          <p-link class="work-pool-card__name" :to="routes.workPool(workPool.name)">
-            {{ workPool.name }}
-          </p-link>
-          <WorkPoolStatusIcon :work-pool="workPool" />
+  <p-card>
+    <div class="work-pool-card">
```

Comment:
fine as is but be nice to do to keep the block class as the root and to also use p-content rather than having custom styling here. 
```
<p-card class="work-pool-card">
  <p-content secondary>
```

Reply:
`<p-card class="work-pool-card">` like just for naming but remove the styles? previously, that class was actually giving styles here

Reply:
Yeah just for naming and remove the styles.

Reply:
I added p-content. but... left the styling on the class. dang it.